### ISSUE TITLE
User Story issue

### ISSUE THREAD

**Issue by dpetravick** [scimma/hop-client#179]
### Description
As a user, I want to follows the documentation in the tutorial and publish a test blob.

But the tutorial l seems to not be accurate for the version of Hop available by pip.

base) ITs-MacBook-Pro:CMB-S4-Hop-play donaldp$ hop --version
hop version 0.0.5
(base) ITs-MacBook-Pro:CMB-S4-Hop-play donaldp$ hop publish --format BLOB kafka://kafka.scimma.org/cmb-s4-fabric-tests.phase-one-testing   blob.txt
usage: hop [-h] [--version] <command> ...
hop: error: unrecognized arguments: --format

Publish directions are here: 
https://github.com/scimma/hop-client/wiki/Tutorial:-using-hop-client-with-the-SCiMMA-Hopskotch-server

I can publish something based on the model shell commands in the tutorial.

### Tasks/Items:
- [ ] First item
- [ ] Second item
- [ ] ...
- [ ] ...
- [ ] ...

**Comment by dpetravick** [comment]
also check this: https://hop-client.readthedocs.io/en/latest/user/commands.html

**Comment by cnweaver** [comment]
I don't think we ever published a hop version 0.0.5? The current version should be 0.5.0, which should support `--format` (and it's a bug for us to fix if it rejects it). However, it seems to work for me with a fresh copy installed from `pip`:
```
$ hop version                                                                                                                                                                                                       
hop-client==0.5.0
adc_streaming==2.0.0
confluent_kafka==1.8.2
librdkafka==1.8.2
$ hop publish --format BLOB kafka://dev.hop.scimma.org/delete.t1 del.txt                                                                                                                                            
2022-04-20 12:45:30,938 | hop : INFO : connecting to kafka://dev.hop.scimma.org
2022-04-20 12:45:30,953 | hop : INFO : publishing to topic: delete.t1
2022-04-20 12:45:30,954 | hop : INFO : publishing messages to stream
2022-04-20 12:45:30,954 | hop : INFO : closing connection
```
